### PR TITLE
CMP-1945 1.2.1 FIO Release Note

### PR DESCRIPTION
**Pull request by GroceryBoyJr**
Version(s):
4.9+

Issue:
https://issues.redhat.com/browse/CMP-1945

Link to docs preview:
[FIO 1.2.1](https://58525--docspreview.netlify.app/openshift-enterprise/latest/security/file_integrity_operator/file-integrity-operator-release-notes.html#file-integrity-operator-release-notes-1-2-1)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by sheriff-rh — COMMENTED** [review]
Looks good!

**Review by Vincent056 — APPROVED** [review]
/lgtm

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc [hunk 1]
```diff
@@ -15,6 +15,13 @@ For an overview of the File Integrity Operator, see xref:../../security/file_int
 
 To access the latest release, see xref:../../security/file_integrity_operator/file-integrity-operator-updating.adoc#olm-preparing-upgrade_file-integrity-operator-updating[Updating the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-1-2-1"]
+== OpenShift File Integrity Operator 1.2.1
+
+The following advisory is available for the OpenShift File Integrity Operator 1.2.1:
```
Comment (rhmdnd) [inline security/file_integrity_operator/file-integrity-operator-release-notes.adoc:21]: For releases where we're not shipping any new FIO code, but just updating the container dependencies, do we typically call that out in the release notes?

Comment (sheriff-rh) [reply security/file_integrity_operator/file-integrity-operator-release-notes.adoc:21]: We have not done it in the past, like [FIO 1.0.0](https://docs.openshift.com/container-platform/4.12/security/file_integrity_operator/file-integrity-operator-release-notes.html#file-integrity-operator-release-notes-1-0-0). But we could add a line below that says "This release includes updated container dependencies." ?

**Comment by sheriff-rh** [comment]
Removing the merge label until the errata is live.

**Review by rhmdnd — COMMENTED** [review]
/lgtm

Thanks, James!

**Comment by sheriff-rh** [comment]
Errata is live; merging.

**Pull request merged**